modules/internal/ChapelArray.chpl

```diff
@@ -1473,6 +1485,98 @@ module ChapelArray {
     }
   }  // record _domain
 
+  proc __getActualInsertPts(d, inds, 
```

Comment:
I'm not so sure this function belongs in ChapelArray. At least it needs a comment indicating it's a helper method for DefaultSparse/CSR.